packages/carbon-addons-boomerang-react/src/components/FeatureNavTabs/FeatureNavTabs.stories.js

```diff
@@ -17,4 +17,16 @@ storiesOf('FeatureNavTabs', module).add('default', () => {
       </Tabs>
     </Router>
   );
+}).add('loading', () => {
+  return (
+    <Router history={createMemoryHistory({ initialEntries: ['/'] })}>
+      <Tabs>
+        <Tab label="Services" to="/services" loading />
```

Comment:
@LucasGrimauth can you make this `isLoading` and I am thinking it would be good to add it to the FeatureHeader in some way. Maybe just the title and subtitle?